buildpacks/dotnet/src/dotnet_buildpack_configuration.rs

```diff
@@ -14,12 +15,16 @@ pub(crate) enum DotnetBuildpackConfigurationError {
     VerbosityLevel(ParseVerbosityLevelError),
 }
 
-impl TryFrom<&libcnb::Env> for DotnetBuildpackConfiguration {
-    type Error = DotnetBuildpackConfigurationError;
-
-    fn try_from(env: &libcnb::Env) -> Result<Self, Self::Error> {
+impl DotnetBuildpackConfiguration {
```

Comment:
(optional) Should the env vars + new project.toml support be listed under the Configuration section in the README? (Since the env vars aren't mentioned there currently either, I'm fine with adding docs later in a separate PR)